internal/sasl/sasl.go

```diff
@@ -26,7 +26,8 @@ import (
 	"unsafe"
 )
 
-type saslStepper interface {
+// Stepper interface for saslSession
+type Stepper interface {
```

Comment:
Intentionally exported?

Reply:
yes, as it is returned by `New(username, password, mechanism, service, host string)` method which is exported ( [sasl.go](https://github.com/globalsign/mgo/blob/master/internal/sasl/sasl.go#L53) ) but maybe it shouldn't be ? I'm not familiar with this part of the driver, so can't tell...

Reply:
I agree the interface should be exported if returned from New.